recipes/numba/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "numba" %}
+{% set version = "0.36.1" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a3b7540713c39f4d7a8d3b18cd49249c6356a35ee1ab5e15a89f7cf970004f0a" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - llvmlite ==0.21.0
+    - numpy 1.10.*
```

Comment:
Forgot about this. There is no `numpy 1.10` for `python 3.6` on Windows. If keeping `numpy 1.10` here is important, would add a selector for `# [win and py<36]` on this and the corresponding `run` line. Then would add the following <code>numpy 1.11.* &nbsp;# [win and py<36]</code> for `build` and similar for `run` except with `>=`. This way `numpy 1.11` can be used with Python 3.6 on Windows. Alternatively can just bump all of the `numpy` versions to 1.11 and then no selectors are needed.

Reply:
Went ahead and just changed the minimum to `numpy 1.11`. If that doesn't work, would be curious to know more about where you would like to use `numpy 1.10`.